script.js

```diff
@@ -1,25 +1,237 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  const episodeCards = makePageForEpisodes(allEpisodes);
-  document.body.append(...episodeCards);
+const showDropDownMenu = document.querySelector('#selectShow');
+let EpisodeDropDown = document.querySelector('#selectEpisode')
+const input = document.querySelector('#input');
+const readMore = document.querySelector('#readMore');
+const home = document.querySelector('#home');
+
+let selectedShow = 1;
+let selectedEpisode = "";
+let SearchTerm = "";
+
+// fetch Api function to fetch the shows
+async function getAllShows() {
+  return await fetch("https://api.tvmaze.com/shows").then((data) => {
+    return data.json();
+  })
+}
+// fetch Api function to fetch the Episodes of selected show
+async function getAllEpisodes() {
+  const rootElem = document.querySelector('#root');
```

Comment:
Are these two variables being used within the function?

Reply:
No, they are not. I have removed them.